vertx-sql-client-template/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,332 @@
+= Client SQL templates
```

Comment:
SQL Client templates

Reply:
we might actually rename this to simply "SQL templates"